### PR TITLE
Update baj-concordia.js

### PR DESCRIPTION
**Pull request by thiefofsanity**
David can't spell. 🙄🙄So this fixes the spelling mistake. 

Delightful commentary had to be cut because people who will not be named were too pedantic to keep it. 🙄🙄🙄

### CODE REVIEW

File: baj-concordia.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ function updateColours() {
     changeColour(index, item);
   });
 }
+  // David can't spell. 🙄🙄
```
Comment (thymepuns) [inline baj-concordia.js]:
```suggestion
```

File: baj-concordia.js [hunk 2]
```diff
@@ -123,6 +124,9 @@ function addNotifications() {
       }
     }
   });
+ 
+   // So this fixes the spelling mistake. 🤞
```
Comment (thymepuns) [inline baj-concordia.js]:
```suggestion
```

File: baj-concordia.js [hunk 3]
```diff
@@ -165,6 +169,8 @@ function main() {
   };
 }
 
+   // While also offering delightful commentary. 😈😈😈😈😈
```
Comment (thymepuns) [inline baj-concordia.js]:
```suggestion
```

**Review by thymepuns — CHANGES_REQUESTED** [review]
Thank you for the PR. Please move your code comments into the PR comments instead.

**Comment by thiefofsanity** [comment]
Well, that's just rude! What about the other emoji? Why are they allowed?

**Review by thymepuns — APPROVED** [review]
👍 LGTM

**Pull request merged**